js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeLeftPanel.tsx

```diff
@@ -65,38 +65,49 @@ export const AutomaterializeLeftList = (props: ListProps) => {
         const isSelected = selectedEvaluation?.evaluationId === evaluation.evaluationId;
         if (evaluation.__typename === 'no_conditions_met') {
           return (
-            <EvaluationListItem
-              key={`skip-${evaluation.evaluationId}`}
-              onClick={() => {
-                onSelectEvaluation(evaluation);
-              }}
-              $selected={isSelected}
-            >
-              <Box flex={{direction: 'column', gap: 4}} style={{width: '100%'}}>
-                <div>
-                  {evaluation.startTimestamp ? (
-                    evaluation.amount === 1 ? (
-                      '1 evaluation'
-                    ) : (
-                      `${compactNumber(evaluation.amount)} evaluations`
-                    )
-                  ) : (
-                    <>
-                      {evaluation.endTimestamp === 'now' ? (
-                        'Before now'
+            <>
```

Comment:
The root node returned from the `map` is the element that needs the key.

If you do keep the message as part of a child in this list, you'll need to move the `key` prop up to the fragment wrapper, e.g. `<React.Fragment key={...}>`.

Reply:
good to know!